signing_service/signing_service/signing_service.py

```diff
@@ -35,17 +44,29 @@ class SigningService:
         'socket',
         'was_sigterm_caught',
         'concent_public_key',
+        'signing_service_private_key',
+        'signing_service_public_key'
     )
 
-    def __init__(self, host, port, initial_reconnect_delay, concent_public_key) -> None:
+    def __init__(
+        self,
+        host: str,
+        port: int,
+        initial_reconnect_delay: int,
+        concent_public_key: bytes,
+        signing_service_private_key: bytes,
+    ) -> None:
         assert isinstance(host, str)
         assert isinstance(port, int)
         assert isinstance(initial_reconnect_delay, int)
         assert isinstance(concent_public_key, bytes)
+        assert isinstance(signing_service_private_key, bytes)
```

Comment:
There seems to be no default value for `signing_service_private_key`, and there should be unless it's not optional argument.
```
./signing_service.sh 127.0.0.1 'yCDaoFX/lgp26x88USJbe8OCkLfkRPcZ8CqjX1cj4tatObo2qVjAa51Mimyurp1kLWj3Z6HqOXY0wxxsagmn4Q=='
```
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/jakub/Documents/work/projects/concent/signing_service/signing_service/signing_service.py", line 334, in <module>
    arg_signing_service_private_key,
  File "/home/jakub/Documents/work/projects/concent/signing_service/signing_service/signing_service.py", line 64, in __init__
    assert isinstance(signing_service_private_key, bytes)
AssertionError
```

Reply:
@cameel @kbeker This is caused by setting default on arguments which can be retrieved in 3 ways (console, file, env), and not adding `required=True` to group definition. By default code is trying to get variable from env, and when it does not exist, it fails. I suggest removing default and adding `required=True` to group definition if you both agree.

Reply:
Sounds fine to me. The option specifying that we should get the value from a variable will still be optional in that case, right?

Reply:
It will be `required` to chose exactly one of three ways of passing variable (console, file, env variable).